crates/cheatcodes/spec/src/vm.rs

```diff
@@ -36,6 +36,24 @@ interface Vm {
         RecurrentPrank,
     }
 
+    /// The kind of account access that occurred.
+    enum AccountAccessKind {
+        /// The account was called.
+        Call,
+        /// The account was called via delegatecall.
+        DelegateCall,
+        /// The account was called via callcode.
+        CallCode,
+        /// THe account was called via staticcall.
+        StaticCall,
+        /// THe account was created.
```

Comment:
Just a typo on the capital H
```suggestion
        /// The account was called via staticcall.
        StaticCall,
        /// The account was created.
```